scripts/components/RepairRequestForm.jsx

```diff
@@ -73,32 +77,35 @@ var RepairRequestForm = React.createClass({
   render() {
     var { description, dataUri } = this.state;
     var errorMessage;
+    var standardActions = [
+      { text: 'Cancel' },
+      { text: 'Submit', onTouchTap: this.onSubmit, ref: 'submit' }
```

Comment:
Try not to use `Submit` as the action for things, as it conveys very little meaning (especially so as it's the default name for any form ever created). Instead describe what action the button performs. This comment form I am currently writing in on Github is a good example. The action says `Comment`, it does not say `Submit`. Similarly, for lodging the repair request say `Lodge Repair Request`, or something like that. It's a nice usability win, and it's very simple to implement once you are in the habit of actually describing what your interface does.